components/Navbar.tsx

```diff
@@ -1,149 +1,325 @@
+          display={{ base: "flex", md: "none" }}
+        >
+          {!!props.session && (
+            <IconButton
+              onClick={onToggle}
+              icon={isOpen ? <MdClose /> : <MdMenu />}
+              variant={"ghost"}
+              aria-label={"Toggle Navigation"}
+            />
           )}
-          {props.hasSession && (
-            <Button
-              variant="outlined"
-              color="info"
-              sx={{ backgroundColor: "white" }}
-              onClick={() => {
-                signOut({
-                  redirect: false,
-                });
-                navigateToAuthPage();
-              }}
+        </Flex>
+        <Flex flex={{ base: 1 }} justify={{ base: "center", md: "start" }}>
+          <Link href={Routes.DASHBOARD}>
+            <Text
+              textAlign={useBreakpointValue({ base: "center", md: "left" })}
+              fontFamily={"heading"}
+              color={useColorModeValue("gray.800", "white")}
+              as='b'
+              textDecoration='none'
             >
-              logout
-            </Button>
-          )}
-          {!props.hasSession && (
+              OnLaunch &#128640;
+            </Text>
+          </Link>
+
+          <Flex display={{ base: "none", md: "flex" }} ml={10}>
+            {!!props.session && <DesktopNav />}
+          </Flex>
+        </Flex>
+
+        <Stack
+          flex={{ base: 1, md: 0 }}
+          justify={"flex-end"}
+          direction={"row"}
+          spacing={6}
+        >
+          {!props.session && (
             <Button
-              variant="outlined"
-              color="info"
-              sx={{ backgroundColor: "white" }}
+              as={"a"}
+              fontSize={"sm"}
+              fontWeight={400}
+              variant={"link"}
               onClick={() => navigateToAuthPage()}
+              href={"#"}
             >
-              login
+              Sign In
             </Button>
           )}
-        </Toolbar>
-      </AppBar>
-      <Box component="nav">
-        <Drawer
-          container={undefined}
-          variant="temporary"
-          open={mobileOpen}
-          onClose={handleDrawerToggle}
-          ModalProps={{
-            keepMounted: true, // Better open performance on mobile.
-          }}
-          sx={{
-            display: { xs: "block", sm: "none" },
-            "& .MuiDrawer-paper": {
-              boxSizing: "border-box",
-              width: drawerWidth,
-            },
-          }}
-        >
-          {drawer}
-        </Drawer>
-      </Box>
+          {!!props.session && (
+            <Menu>
+              <MenuButton
+                as={Button}
+                rounded={"full"}
+                variant={"link"}
+                cursor={"pointer"}
+                minW={0}
+              >
+                <Avatar
+                  size={"sm"}
+                  src={"https://avatars.githubusercontent.com/u/38766452"}
```

Comment:
Hardcoding your profile image is not the best way. Use a different placeholder or a static example image instead.